robolectric-shadows/shadows-maps/src/main/java/org/robolectric/shadows/maps/ShadowGeocoder.java

```diff
@@ -69,7 +71,7 @@
 
   private Address makeAddress() {
     Address address = new Address(Locale.getDefault());
-    Shadows.shadowOf(address).setSimulatedHasLatLong(hasLatitude, hasLongitude);
+    ((ShadowAddress) ShadowExtractor.extract(address)).setSimulatedHasLatLong(hasLatitude, hasLongitude);
```

Comment:
Why do you need this change? Seems you should prefer the autogenerated Shadows.shadowOf()?